pyspades/server.py

```diff
@@ -249,9 +275,15 @@ def update_network(self):
         world_update = loaders.WorldUpdate()
         # we only want to send as many items of the player list as needed, so
         # we slice it off at the highest player id
-        world_update.items = items[:highest_player_id+1]
-        self.broadcast_contained(world_update, unsequenced=True)
-
+        world_update.items = items[:max(self.players)+1]
+        if for_spectators:
```

Comment:
I'm not sure I like the idea of not being able to configure this at all. Ideally you'd be able to configure the frequency, or at least set it with one variable.